src/test/scala/chiselTests/reflect/DataMirrorSpec.scala

```diff
@@ -245,4 +245,24 @@ class DataMirrorSpec extends ChiselFlatSpec {
     DataMirror.getLayerColor(foo.c) should be(Some(A))
   }
 
+  "moduleIOs" should "return an in-progress module's IOs" in {
```

Comment:
super nit you changed the name of this function and maybe we should have changed this string

Reply:
thanks, i wish we could push NFC changes to main.